File: Strings/notes.c
// Evelyn Chennault, Notes C
#include <stdio.h>

char subject[50];

int main(void){
    //printf("What class are you in?\n");
    //scanf("%s", subject);
    fgets(subject, sizeof(subject), stdin);
    printf("You are in %s, that is a cool class.");
    return 0;
}

File: Strings/Notes.py
# Evelyn Chnenault, Strings Notes Python

# strings are a data type that holds any info surrounded by quotation marks "" ''

# note = "Ms. LaRose's class"

name = input("What is your first name?\n").strip().capitalize()

print(f"Hello, {name}. Welcome to my program!")

# print("This is your name: "+ name)

sentence = "The quick brown fox jumps over the lazy dog"

# print(len(sentence))
# print(sentence[4])
start = sentence.find("brown")
length = len("brown fox")
print(sentence[start:start+length])